src/Connection/Connection.php

```diff
@@ -15,7 +15,7 @@
 use Bunny\Exception\ClientException;
 use Gamee\RabbitMQ\Connection\Exception\ConnectionException;
 
-final class Connection
+class Connection
```

Comment:
Wouldn't it be better to add a interface rather than removing the `final` keyword?

Reply:
ok